src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilerDiagnosticAnalyzer.cs

```diff
@@ -40,8 +40,6 @@ public sealed override void Initialize(AnalysisContext context)
                 c.RegisterSyntaxTreeAction(analyzer.AnalyzeSyntaxTree);
                 c.RegisterSemanticModelAction(CompilationAnalyzer.AnalyzeSemanticModel);
             });
-
-            context.RegisterCompilationAction(CompilationAnalyzer.AnalyzeCompilation);
```

Comment:
since it is probably most stable way to de-dup build and live for compiler errors for now.